tests/utils/datatype_tests.py

```diff
@@ -55,6 +56,50 @@ def test_tile_transform(self):
         with self.assertRaises(ValueError):
             boolean.transform_value_for_tile(None)
 
+class GeoJsonDataTypeTest(ArchesTestCase):
+
+    def test_validate_reduce_byte_size(self):
+        f = open("tests/fixtures/problematic_excessive_vertices.geojson")
+        geom = json.load(f)
+        geom_datatype = DataTypeFactory().get_instance("geojson-feature-collection")
+        errors = geom_datatype.validate(geom)
+        self.assertEqual(len(errors), 0)        
+
+
+    def test_validate_bbox(self):
+        settings.DATA_VALIDATION_BBOX = [(
+              12.948801570473677,
+              52.666192057898854
+            ),
+            (
+              12.948801570473677,
+              52.26439571958821
+            ),
+            (
+              13.87818788958171,
+              52.26439571958821
+            ),
+            (
+              13.87818788958171,
+              52.666192057898854
+            ),
+            (
+              12.948801570473677,
+              52.666192057898854
+            )]
+
+        geom_datatype = DataTypeFactory().get_instance("geojson-feature-collection")
+
+        with self.subTest():
```

Comment:
If you give the subTest a label with a kwarg, you can tell the two failures apart.